src/stories/SectionPagination.stories.js

```diff
@@ -42,10 +40,12 @@ export function LastPage() {
   }
 }
 
+// this story uses the generateLinkCallback prop
+// to generate the links in the library-website-nuxt format instead of the default format
 export function WithPagesAndCurrentPage() {
   return {
     components: { SectionPagination },
-    template: '<section-pagination :pages="23" :initialCurrentPage="4" />',
+    template: '<section-pagination :pages="23" :initialCurrentPage="4" :generateLinkCallback="(x) => { return `/page/` + x }"/>',
```

Comment:
i think this should be return `?page=` + x?

Reply:
shouldnt it be query param?

Reply:
I based the pattern on the way the data looks for the default stories ( /page/1, /page/10). But I will also include a story that uses the queryparam in the path to make sure that works without any more tinkering. I see the live implementation on the library site looks like 'https://www.library.ucla.edu/search-site?q=new&from=20' so I'll try to implement that

Reply:
I tweaked the story to use the callback to send the link in the same format as the library site